third_party/accessibility/BUILD.gn

```diff
@@ -45,50 +45,71 @@ if (enable_unittests) {
   executable("accessibility_unittests") {
     testonly = true
 
-    sources = [
-      "ax/ax_enum_util_unittest.cc",
-      "ax/ax_event_generator_unittest.cc",
-      "ax/ax_node_data_unittest.cc",
-      "ax/ax_node_position_unittest.cc",
-      "ax/ax_range_unittest.cc",
-      "ax/ax_role_properties_unittest.cc",
-      "ax/ax_table_info_unittest.cc",
-      "ax/ax_tree_unittest.cc",
-      "ax/test_ax_node_helper.cc",
-      "ax/test_ax_node_helper.h",
-      "ax/test_ax_tree_manager.cc",
-      "ax/test_ax_tree_manager.h",
-    ]
+    public_configs = [ ":accessibility_config" ]
 
-    sources += [
-      "ax/platform/ax_platform_node_base_unittest.cc",
-      "ax/platform/ax_platform_node_unittest.cc",
-      "ax/platform/ax_platform_node_unittest.h",
-      "ax/platform/ax_unique_id_unittest.cc",
-      "ax/platform/test_ax_node_wrapper.cc",
-      "ax/platform/test_ax_node_wrapper.h",
-    ]
+    # TODO(cbracken): make this block unconditional once Windows node support
+    # is landed.
+    if (is_mac) {
+      sources = [
+        "ax/ax_enum_util_unittest.cc",
+        "ax/ax_event_generator_unittest.cc",
+        "ax/ax_node_data_unittest.cc",
+        "ax/ax_node_position_unittest.cc",
+        "ax/ax_range_unittest.cc",
+        "ax/ax_role_properties_unittest.cc",
+        "ax/ax_table_info_unittest.cc",
+        "ax/ax_tree_unittest.cc",
+        "ax/test_ax_node_helper.cc",
+        "ax/test_ax_node_helper.h",
+        "ax/test_ax_tree_manager.cc",
+        "ax/test_ax_tree_manager.h",
+      ]
 
-    sources += [
-      "base/logging_unittests.cc",
-      "base/string_utils_unittest.cc",
-    ]
+      sources += [
+        "ax/platform/ax_platform_node_base_unittest.cc",
+        "ax/platform/ax_platform_node_unittest.cc",
+        "ax/platform/ax_platform_node_unittest.h",
+        "ax/platform/ax_unique_id_unittest.cc",
+        "ax/platform/test_ax_node_wrapper.cc",
+        "ax/platform/test_ax_node_wrapper.h",
+      ]
+
+      deps = [
+        ":accessibility",
+        ":accessibility_fixtures",
+        "//flutter/testing",
+        "//flutter/testing:dart",
+      ]
+    } else if (is_win) {
+      sources = [
+        "base/win/dispatch_stub.cc",
+        "base/win/dispatch_stub.h",
+        "base/win/scoped_bstr_unittest.cc",
+        "base/win/scoped_safearray_unittest.cc",
+        "base/win/scoped_variant_unittest.cc",
+      ]
+
+      deps = [
+        ":accessibility_fixtures",
+        "//flutter/third_party/accessibility/base",
+        "//flutter/testing",
+        "//flutter/testing:dart",
+      ]
+    }
 
     sources += [
```

Comment:
This needs to be in both blocks or in an `if (is_mac || is_win) {` block to avoid making other platforms sad.

Reply:
Done